src/NexusMods.App.UI/Pages/MyGames/MyGamesViewModel.cs

```diff
@@ -49,7 +49,10 @@ public class MyGamesViewModel : APageViewModel<IMyGamesViewModel>, IMyGamesViewM
         this.WhenActivated(d =>
             {
                 var loadouts = loadoutRepository.Observable
-                    .ToObservableChangeSet();
+                    .ToObservableChangeSet()
+                    .Filter(loadout => loadout.IsVisible())
+                    .GroupOn(loadout => loadout.Installation.LocationsRegister[LocationId.Game])
+                    .Transform(group => group.List.Items.First());
```

Comment:
This can probably just be another filter:

```suggestion
                    .Filter(loadout => loadout.Installation.LocationsRegister[LocationId.Game]);
```

Reply:
Suggested code doesn't work syntactically as Filter requires a bool result from the Func.
Semantically I'm also not convinced a Filter can do what we want here, which is to only take one loadout per game install (distinct by Installation).

Reply:
```csharp
.DistinctValues(loadout => loadout.Installation.LocationsRegister[LocationId.Game]
```

Reply:
Huh, that's an odd behavior.